src/applications/check-in/components/LinkTranslation.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
+
+function LinkTranslation({ children, href, hrefLang }) {
```

Comment:
This is picky but this name doesn't really describe the component's function to me. Maybe even just `ExternalLink`? I could also see getting asked to add an icon or other styling/functionality down the road. What do you think?

Reply:
`LinkTranslation` definitely has some room for improvement on the name. This has been updated to use `ExternalLink `now. I think that describes the component's purpose much better. Thanks!

Reply:
nice!